packages/tools/src/tools/segmentation/LabelmapBaseTool.ts

```diff
@@ -407,10 +460,8 @@ export default class LabelmapBaseTool extends BaseTool {
       this.getOperationData(element),
       StrategyCallbacks.AcceptPreview
     );
-    this._previewData.isDrag = false;
     this._previewData.preview = null;
-    // Store the edit memo too
-    this.doneEditMemo();
+    this._previewData.isDrag = false;
```

Comment:
This should reset the isDrag to whatever it was before this - which could still be isDrag depending on how often memos are stored.  That is definitely on mouse up - you can test the wrong isDrag setting by checking the un/re do functionality for selecting a region, moue up, selecting more region, mouse up, removing a region, mouse up, then accept.